scalalib/test/src/mill/scalalib/TestModuleUtilTests.scala

```diff
@@ -254,6 +254,7 @@ object TestModuleUtilTests extends TestSuite {
         )
       )
     }
+    // format: off
```

Comment:
I did this because it keeps fucking up the format inside this function. format didn't seem to turn off on a llne based usage.

Reply:
Sounds good, maybe something to do with the XML literals which are pretty obscure and mayb the formatter doesn't work well with